### PR TITLE
Lasb 513 add cracked trial

### PR DESCRIPTION
**Pull request by chris-groves**
## What

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=454&projectKey=LASB&modal=detail&selectedIssue=LASB-513

Enable a 'cracked ineffective trial' to be recorded against a hearing.

## Checklist

Before you ask people to review this PR:

- [x] Tests and linters should be passing
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/admin/hearings/_form.html.erb [hunk 1]
```diff
@@ -15,47 +15,62 @@
...
+
   <div class="field">
     <%= form.label :isBoxHearing %>
     <%= form.check_box :isBoxHearing %>
   </div>
 
+  <h3>Cracked Ineffective Trial</h3>
+
+  <div class="field">
+    <%= form.select(:cracked_ineffective_trial_id, CrackedIneffectiveTrial.all.collect { |cit| [cit.description, cit.id] }, include_blank: true) %>
```
Comment (arthurashman) [inline app/views/admin/hearings/_form.html.erb:71]: Does this dropdown work for you locally?
CrackedIneffectiveTrial.all is an empty array for me

Comment (laurentqro) [reply app/views/admin/hearings/_form.html.erb:71]: @arthurashman running `rake task mock:demodata:load` will create a cracked trial record in the database, and the dropdown will have it.

https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/lib/tasks/demodata.rake

**Review by laurentqro — APPROVED** [review]
👍🏻 after conflicts have been resolved

**Review by laurentqro — APPROVED** [review]

File: db/seeds.rb [hunk 2]
```diff
@@ -2,4 +2,65 @@
...
+CrackedIneffectiveTrial.create(code: "Q1", description: "Defence not ready: disclosure problems", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "Q2", description: "Defence not ready: specify in comments (inc. no instructions)", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "Q3", description: "Defence asked for additional prosecution witness to attend", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "R", description: "Defence witness absent", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S1", description: "Defendant absent - did not proceed in absence (judicial discretion)", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S2", description: "Defendant ill or otherwise unfit to proceed", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S3", description: "Defendant not produced by PECS", reason_type: "ineffective")
+CrackedIneffectiveTrial.create(code: "S4", description: "Defendant absent â€“ unable to proceed as Defendant not notified of place and time of hearing", reason_type: "ineffective")
```
Comment (arthurashman) [inline db/seeds.rb:36]: Could you replace the â€“ with a -?  The google sheet had â€“ but I think we're safe to switch to a - 
There are a couple in this data

**Review by arthurashman — APPROVED** [review]
Looks good to me

**Pull request merged**